addon/services/observer-manager.js

```diff
@@ -23,4 +23,9 @@ export default class ObserverManagerService extends Service {
   willDestroy() {
     this._admin.destroy();
   }
+
+  removeElement(element) {
+    this._admin.elementRegistry.removeElement(element);
```

Comment:
`elementRegistry` is private. Are you sure we can access it?

https://github.com/snewcomer/intersection-observer-admin/blob/master/src/index.ts#L22